test/builtin/arithmetic/test_basic.py

```diff
@@ -153,8 +153,8 @@ def test_multiply(str_expr, str_expected, msg):
         ("a  b  DirectedInfinity[q]", "a b (q Infinity)", ""),
         # Failing tests
         # Problem with formatting. Parenthezise are missing...
-        #        ("a  b  DirectedInfinity[-I]", "a b (-I Infinity)",  ""),
-        #        ("a  b  DirectedInfinity[-3]", "a b (-Infinity)",  ""),
+        ("a  b  DirectedInfinity[-I]", "a b (-I Infinity)", ""),
+        ("a  b  DirectedInfinity[-3]", "a b (-Infinity)", ""),
```

Comment:
Same thing here.

Reply:
In any case, the test seems to be right. The problem is that if we activate it, it does not pass it. I can revert the change, but it will not have any effect until the underlying problem gets fixed, and the tests get activated.